src/Stratis.Bitcoin/FullNode.cs

```diff
@@ -241,12 +241,20 @@ private void StartPeriodicLog()
                 this.LastLogOutput = stats;
 
                 return Task.CompletedTask;
-            },
-            this.nodeLifetime.ApplicationStopping,
-            repeatEvery: TimeSpans.FiveSeconds,
-            startAfter: TimeSpans.FiveSeconds);
+            }, this.nodeLifetime.ApplicationStopping, repeatEvery: TimeSpans.FiveSeconds, startAfter: TimeSpans.FiveSeconds);
 
             this.Resources.Add(periodicLogLoop);
+
+            IAsyncLoop periodicBenchmarkLoop = this.AsyncLoopFactory.Run("PeriodicBenchmarkLog", (cancellation) =>
+            {
+                string benchmark = this.nodeStats.GetBenchmark();
+
+                this.logger.LogInformation(benchmark);
+
+                return Task.CompletedTask;
+            }, this.nodeLifetime.ApplicationStopping, repeatEvery: TimeSpan.FromSeconds(17), startAfter: TimeSpan.FromSeconds(17));
+
+            this.Resources.Add(periodicBenchmarkLoop);
```

Comment:
We want to get it of this list of resources, it is not helping if you're adding to it

Reply:
It will be done within this week, don't worry about it.

If Id fix all those small issues the PR would be huge and you'd ask me to split it in parts.

Reply:
you misunderstood, what i'm saying is don't use something you will remove in next PR, you are just creating more work for yourself